app/src/tests/component-tests/OrderBookSnapshot.test.js

```diff
@@ -77,3 +77,49 @@ describe('date and time picker functionality', () => {
         expect(selectedDateNano).toEqual(DATE_VALUE+TIME_VALUE);
     });
 });
+
+describe('navigating by message functionality', () => {
+    let mount, shallow;
+    const getOrderBookPricesSpy = jest.spyOn(OrderBookService, 'getOrderBookPrices').mockImplementation((instrument, timestamp) => Promise.resolve({ data: () => [] }));
+    const getOrderBookPricesByMessageOffsetSpy = jest.spyOn(OrderBookService, 'getOrderBookPricesByMessageOffset').mockImplementation((instrument, timestamp, offset) => Promise.resolve({ data: () => [] }));
```

Comment:
If you want to start testing your implementation without the backend, you could have an orderbook structure here where you remove most of the price levels in the asks/bids response.